components/expenses/ExpensePayeeDetails.js

```diff
@@ -327,6 +327,7 @@ ExpensePayeeDetails.propTypes = {
       }),
     }),
     payeeLocation: PropTypes.shape({
+      // TODO: fix
```

Comment:
?

Reply:
Leftovers.. removed! (no longer needed to fix after deciding to keep the existing location shape)

Reply:
Leftovers.. removed! (no longer needed to fix after deciding to keep the existing location shape)